Dockerfile

```diff
@@ -68,22 +64,18 @@ RUN apk add --no-cache \
     python3 \
     python3-dev \
     py3-pip \
-    shellcheck
+    shellcheck \
+    libc6-compat
 
 # Install pre-commit framework
 RUN pip3 install --ignore-installed pre-commit==$PRECOMMIT_VERSION
 
 # Install checkov
 RUN pip3 install --ignore-installed checkov==$CHECKOV_VERSION
 
-# Download Terraform, verify checksum and install to bin dir
-RUN wget \
-    $TERRAFORM_URL \
-    $TERRAFORM_CHECKSUM_URL && \
-    sed -i '/terraform_.*_linux_amd64.zip/!d' $TERRAFORM_CHECKSUM && \
-    sha256sum -cs $TERRAFORM_CHECKSUM && \
-    unzip $TERRAFORM_ARCHIVE -d /usr/local/bin && \
-    rm -f $TERRAFORM_ARCHIVE $TERRAFORM_CHECKSUM
+# Install tfswitch and latest stable Terraform release
+RUN wget -O- -nv https://raw.githubusercontent.com/warrensbox/terraform-switcher/release/install.sh | sh
+RUN tfswitch --latest-stable $TF_VERSION
```

Comment:
this should be done at run timw now.. else the container runs with a random version..

Reply:
it's done at run time of `$TF_VERSION` is set at runtime. If you prefer, we can remove the `--latest-stable` from the Dockerfile to enforce that it will install exactly the version that is set in the `TF_VERSION` build arg. Please note that we have both, `TF_VERSION` environment variable to dynamically set the version on runtime and `TF_VERSION` build arg to pre-install a Terraform version at build-time.